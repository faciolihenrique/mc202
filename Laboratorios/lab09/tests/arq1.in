# Teste com um aluno
# 
i 123456  João da Silva
n
v
p
l
x
