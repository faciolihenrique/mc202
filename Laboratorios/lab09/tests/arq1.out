Ação: # Teste com um aluno
Ação: # 
Ação: i 123456 João da Silva
Ação: n
Número de elementos: 1.
Ação: v
Fila válida
Ação: p
 123456 João da Silva
Ação: l
Ação: x
Processamento terminado
